projects/getting_started/src/main.c

```diff
@@ -0,0 +1,28 @@
+// Turns the LED on and sits forever
```

Comment:
This this project should not be part of this pull request. We normally keep our work on seperate branches so when we make a PR we know exactly what is on that branch and what ticket it relates to.